keel-artifact/src/test/kotlin/com/netflix/spinnaker/keel/artifact/ArtifactListenerTests.kt

```diff
@@ -148,7 +162,7 @@ internal class ArtifactListenerTests : JUnit5Minutests {
 
     context("artifact is already registered") {
       before {
-        every { repository.isRegistered("fnord", deb) } returns true
+        every { repository.isRegistered("fnord", DEBIAN) } returns true
```

Comment:
Is this going to break serialization/deserialization anywhere?

Reply:
Nope. Same values, just different variable names.

Reply:
I don't care about the names, just confirming that it serializes the same way

Reply:
Yep, all good.